public_html/app/Controllers/Countries.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+
+namespace VFramework\Controllers;
+
+use VFramework\Api\Client;
+use VFramework\Api\CountriesApi;
+use VFramework\Helpers\UrlHelper;
+use VFramework\Models\AbstractModel;
+use VFramework\Models\Country;
+
+class Countries
+{
+    public function __construct()
+    {
+        $this->model = new Country();
+    }
+
+    /**
+     * @var array
+     */
+    protected $limitedCountries = [];
+
+
+    public function countriesApi(): void
```

Comment:
This function should return something. True if call was successfully send false if not.